components/button/button.css

```diff
@@ -0,0 +1,97 @@
+
+.btn {
+    padding: 0.5rem 1rem;
+    margin: 0.5rem;
+    background: var(--white-color);
+}
+
+.btn-primary {
+    background: var(--primary-color);
+    color: var(--white-color);
+    border-radius: 2.5px;
+    /* opacity: 0.5; */
+}
+
+.btn-success {
+    background: var(--success-color);
+    color: var(--white-color);
+    border-radius: 2.5px;
+}
+
+.btn-warning {
+    background: var(--warning-color);
+    color: var(--white-color);
+    border-radius: 2.5px;
+}
+
+.btn-danger {
+    background: var(--danger-color);
+    color: var(--white-color);
+    border-radius: 2.5px;
+}
+
+.btn-info {
+    background: var(--info-color);
+    border-radius: 2.5px;
+}
+
+.btn-primary-outline {
+    border: 2px solid var(--primary-color);
+    color: var();
+    border-radius: 2.5px;
+}
+
+.btn-primary-outline:hover {
+    background: var(--primary-color);
+    color: var(--white-color);
+}
+.btn-success-outline {
+    border: 2px solid var(--success-color);
+    color: var();
+    border-radius: 2.5px;
+}
+
+.btn-success-outline:hover {
+    background: var(--success-color);
+    color: var(--white-color);
+}
+.btn-warning-outline {
+    border: 2px solid var(--warning-color);
+    color: var();
+    border-radius: 2.5px;
+}
+
+.btn-warning-outline:hover {
+    background: var(--warning-color);
+    color: var(--white-color);
+}
+.btn-danger-outline {
+    border: 2px solid var(--danger-color);
+    color: var();
```

Comment:
Empty variables on lines 70 and 80.

Reply:
Oops, I missed to add the names. Corrected them.